docs/index.html

```diff
@@ -12,7 +12,7 @@ <h1>MA Projects</h1>
                     Vladyslav Yermolin - <a href="./vladyslav.yermolin/index.html">Lecture 5</a>
                 </li>
                 <li>
-                    Serhii Noskov - <a href="./SerhiiNSN/index.html">Final Project</a>
+                    Vladyslav Brusentsov - <a href="./vBrusentsov/about.html">Games Project</a>
```

Comment:
Hey, @vBrusentsov, why do you delete and change so many files of other students or even lecturers?
You should only add your changes, not delete any work of your colleagues.

Please revert these changes.

And here you should add the links to your project without deleting other student's link.
```suggestion
 <li>
                    Serhii Noskov - <a href="./SerhiiNSN/index.html">Final Project</a>
</li>
<li>
                    Vladyslav Brusentsov - <a href="./vBrusentsov/about.html">Games Project</a>
</li>
```